benchmarks/vc_soa/include/benchmark/vc_soa/data_generator.hpp

```diff
@@ -80,8 +80,9 @@ inline void fill_random_matrix(std::vector<matrix_t> &collection) {
 
       for (std::size_t i = 0u; i < matrix_t::rows(); ++i) {
         v[i] = simd_vector_t::Random();
-        m[j] = v;
       }
+
+      m[j] = v;
```

Comment:
Oh, snap... Did you find a bug as part of these updates? 🤔

Reply:
This is part of the random data generator for the benchmarks. And I think it has been setting the same vector multiple times instead of once, so it was just doing some pointless work

Reply:
👍 I saw that this was in the benchmarks. That's why I was also not super worried about it. 😉